packages/lib/src/components/Card/types.ts

```diff
@@ -211,6 +211,7 @@ export interface BrandObject {
     supported: boolean;
     brandImageUrl?: string;
     panLength?: number;
+    paymentMethodVariant?: string;
```

Comment:
Isn't that an array of strings? Is it possible to have more than one value in that array?

Reply:
This is for the single object(s) that get returned in the /binLookup response.
We collect the `paymentMethodVariant` strings and put them into a `paymentMethodVariants` array